docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -814,3 +814,49 @@ files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
 scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
 
 Only `GET` is currently supported.
+
+### 5.3 Upload DICOM file
+```
+PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename/$IsPhantom
+```
+
+Upload a DICOM file that is compressed as a `.zip`,`.tar.gz` or `.tgz`.
+$CandID must be the full name of the file and its extension.
+$IsPhantom must be `'Y'` or `'y'` if it's a phantom file, else `'N'` or `'n'` 
+if not.
+
+Returns JSON data upon success having the form:
+
+```js
+{
+    "success":"Uploaded",
+    "mri_upload_id":"3"
+}
+```
+
+mri_upload_id is the Upload ID for the file.
+
+### 5.4 Process DICOM file
+```
+POST /candidates/$CandID/$VisitLabel/dicoms/$Filename/$IsPhantom
```

Comment:
I suggest `POST /candidates/$CandID/$VisitLabel/dicoms/$Filename/processes` with a payload like 
```js
{
  "process_type": "mri_upload",
  "filename" : $filename
}
```
with a response like 
```js
{
  "process_id": 123,
  "status": "started|running|complete|failed",
  "message": "..."
}
```
Then, `GET /candidates/$CandID/$VisitLabel/dicoms/$Filename/processes/123`
would return 
```js
{
  "process_id": 123,
  "status": "started|running|complete|failed",
  "message": "..."
}
```

Reply:
Great! I will make the changes.

The logic to get the status of the process will be added (since there i hadn't put in yet)